src/sidebar/components/autocomplete-list.js

```diff
@@ -12,6 +30,8 @@ const defaultListFormatter = item => item;
  * used by itself.
  *
  * Modeled after the "ARIA 1.1 Combobox with Listbox Popup"
+ * @template T
```

Comment:
Style nit, can you leave a new line between the description of a function and the `@template`/`@param`  comments